_hardware/04-CompatibilityCharts/05-Misc/index.md

```diff
@@ -27,7 +27,24 @@ has_toc: false
 <h2 align="center">An info dump of obscure hardware support</h2>
 <br>
 
-<p align="center">This section is currently a placeholder, and may even be removed!</p>
+<p align="center">This section includes hardware that doesn't fall into any of the previous categories.</p>
+
+# Fingerprint Sensors
+
+- Fingerprint sensors won't work in macOS.
```

Comment:
This is only partly correct - It wont work on a Hac, right - nethertheless - the issue is not macOS. Apple requires a T1 or T2 chip for Fingerpring readers.